frontend/src/components/NavBar.css

```diff
@@ -0,0 +1,130 @@
+/* Reset styles */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.btn-12,
```

Comment:
btn-12 is not descriptive of the function or the element. Could this be something like 'nav-btn'?
This applies to all mention of btn-12. Quick to change with find and replace.